companies/playmobil-de.json

```diff
@@ -0,0 +1,21 @@
+{
+    "slug": "playmobil-de",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "commerce",
+        "entertainment"
+    ],
+    "name": "geobra Brandstätter Stiftung & Co. KG",
```

Comment:
Shouldn't there be a `runs` for `*playmobil*`? :D

Reply:
Oh yes, ofc, great